zone/gm_commands/parcels.cpp

```diff
@@ -69,36 +68,38 @@ void command_parcels(Client *c, const Seperator *sep)
 		if (arguments < 2) {
 			c->Message(Chat::White, "Usage: #parcels listmemory [Character Name]");
 		}
-		else {
-			if (!player) {
-				c->Message(
-					Chat::White,
-					fmt::format(
-						"Player {} could not be found in this zone.  Ensure you are in the same zone as the player.",
-						player_name
-					)
-						.c_str());
-				return;
-			}
-			else {
-				auto parcels = player->GetParcels();
-				if (parcels.empty()) {
-					c->Message(Chat::White, fmt::format("No parcels could be found for {}", player_name).c_str());
-				}
-				c->Message(Chat::Yellow, fmt::format("Found {} parcels for {}.", parcels.size(), player_name).c_str());
-				for (auto const &p: parcels) {
-					c->Message(
-						Chat::Yellow, fmt::format(
-							"Slot [{:02}] has item id [{:10}] with quantity [{}].",
-							p.second.slot_id, p.second.item_id, p.second.quantity
-						)
-							.c_str());
-				}
-			}
+
+		if (!player) {
+			c->Message(
+				Chat::White,
+				fmt::format(
+					"Player {} could not be found in this zone.  Ensure you are in the same zone as the player.",
+					player_name
+				).c_str()
+			);
+			return;
+		}
+
+		auto parcels = player->GetParcels();
+		if (parcels.empty()) {
+			c->Message(Chat::White,fmt::format("No parcels could be found for {}",player_name).c_str());
```

Comment:
Formatting, I'd put some spaces are the very least.

Reply:
Updated